### ISSUE TITLE
consider using mapconcat

### ISSUE THREAD

**Issue by PengjuXu** [tmtxt/tmtxt-dired-async#2]
in your code there are:
    (dolist (file files)
      (setq command (concat command (shell-quote-argument file) " ")))
example: (defun tda/rsync (dest)

consider mapconcat

**Comment by tmtxt** [comment]
yeah, will update it soon. this package is when I first started with lisp
